lib/services/mediaServicesManagement/package.json

```diff
@@ -14,8 +14,8 @@
     "node",
     "azure"
   ],
-  "main": "./lib/mediaservicesManagementClient.js",
```

Comment:
Thanks for the contribution. Can you also update the version to "1.0.1-preview" in this PR?